packages/vuepress-theme-nimatron/layouts/Layout.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="nimatron">
+    <Sidebar
+      :items="SidebarItems"
+    >
+      <template #top>
+        <slot name="Sidebar-top" />
+      </template>
+      <template #bottom>
+        <slot name="Sidebar-bottom" />
+      </template>
+    </Sidebar>
+    <main class="nimatron--main">
+      <article class="nimatron--article">
+        <transition name="fade">
+            <Content />
+        </transition>
+      </article>
+    </main>
+  </div>
+</template>
+
+
+<script>
+import { resolveSidebarItems } from '../utils'
+
+export default {
+  components: {
+    Sidebar: () => import('../components/Sidebar.vue'),
+    Example: () => import('../global-components/Example.vue'),
```

Comment:
Do we need this global example?

Reply:
Yeah so basically we're registering the Example component for use in the "Layout" layout. Doing this means we can use it within our markdown.